tests/unit/plugins/ovs/test_ovsclients.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import ddt
+import mock
+
+from rally_ovs.plugins.ovs import ovsclients
```

Comment:
Because rally_ovs is a plugin of rally, and this repo is a fork of upstream rally, it's better to put this test case into  openvswitch/ovn-scale-test.

Reply:
@l8huang I would like to do that but I don't know how to tackle it. The unit testing framework (tests.unit.test) is not part of Rally distribution. I'm not sure if Rally supports having tests for plugins that  are distributed separately. Do you know how we could do it?

Reply:
OK, I think we can hack it up and have tests in ovn-scale-test repo.

I was able to run this test when placed in the ovn-scale-test repo by adding the path to a clone of this repo to Python's search path. This way tests in ovn-scale-test can access the `tests` module from this repo.

Let's close this one.